lib/chopperbot/router.ex

```diff
@@ -1,27 +1,25 @@
 defmodule Chopperbot.Router do
   use Plug.Router
-  alias Chopperbot.{Character, Split}
 
-  plug(:match)
-  plug(Plug.Parsers, parsers: [:json, :urlencoded], json_decoder: Jason)
-  plug(:dispatch)
+  alias Chopperbot.MessageBuilder
+
+  @line_message Linex.get_conf(:message)
+
+  plug :match
+  plug Plug.Parsers, parsers: [:json, :urlencoded], json_decoder: Jason
+  plug :dispatch
 
   post "/hello" do
     send_resp(conn, 200, "world")
   end
 
   post "/split" do
-    input = conn.body_params["text"]
-    response = build_response(input)
-
-    body = %{
-      "response_type" => "in_channel",
-      "text" => response
-    }
+    text = conn.body_params["text"]
+    message = MessageBuilder.build(text, for: :slack)
```

Comment:
This becomes a lot cleaner 👍.

Reply:
From the design comment above, maybe is it better to make it something like,

```ex
response = Split.process(text_input, for: slack)
```

Reply:
Cool! 👍